FileGopherServer.cpp

```diff
@@ -218,7 +220,7 @@ FileGopherServer::FileGopherServer(string root, int port)
  */
 FileGopherServer::FileGopherServer()
 {
-    FileGopherServer("/", 70);
+    FileGopherServer("/", 70, show_hidden_files);
```

Comment:
Use `false` as default value here instead of `show_hidden_files`

Reply:
Done